Content.Server/Botany/Systems/BotanySystem.Produce.cs

```diff
@@ -15,10 +15,13 @@ public void ProduceGrown(EntityUid uid, ProduceComponent produce)
         solutionContainer.RemoveAllSolution();
         foreach (var (chem, quantity) in seed.Chemicals)
         {
-            var amount = FixedPoint2.New(quantity.Min);
-            if (quantity.PotencyDivisor > 0 && seed.Potency > 0)
-                amount += FixedPoint2.New(seed.Potency / quantity.PotencyDivisor);
-            amount = FixedPoint2.New(MathHelper.Clamp(amount.Float(), quantity.Min, quantity.Max));
+            var amount = FixedPoint2.Zero;
+            if (seed.Potency > 0)
+            {
+                amount += FixedPoint2.New(MathHelper.Lerp(quantity.Min, quantity.Max, seed.Potency / 50));
+                // Adds an amount, depending on the potency. The maximum is 50 as per RobustHarvest.cs,
```

Comment:
Now the maximum isn't correctly honored. This should clamp at maximum potency. This new denominator should be greater than 50, because otherwise there would be no increase in chemicals despite increases in potency.

Reply:
I have the denominator at 50 because it is the maximum when using Robust Harvest in-game, and I have seen botanists (such as the comments above) talk about 50 as the "practical" maximum. It is possible to get potency above 50, and in my tests the `Lerp` function successfully "extrapolated". (By which I mean, when potency is above 50, there is an increase in chemicals above the "maximum" value.) I think being able to use mutation to get potency/chemicals "above the maximum" is a fun sort of chaos in the game, and does not need to be removed in this PR. Would it be a reasonable compromise to have it clamp at 100 potency (2x"max")?

Reply:
So then `max` is not truly a maximum, and the datafield name should reflect that. Perhaps `robustHarvestMax`, and then un-hard-code the robust harvest limit.

Reply:
These are good ideas, thank you!

Reply:
Did you have a strategy in mind for un-hard-coding the robust harvest limit? I'm unsure how to reference PotencyLimit from the BotanySystem class.